entry.go

```diff
@@ -29,7 +29,13 @@ func (e *Entry) GetUpdatedAt() (updatedAt int64) {
 
 // GetRelationshipIDs will get the associated relationship IDs
 // Note: This will have to be replaced by the including entry if relationships are needed
-func (e *Entry) GetRelationshipIDs() (ids []string) {
+func (e *Entry) GetRelationshipIDs(deprecated string) {
```

Comment:
Why the signature change? Can we still support this function w/ it's old signature at the same time, or is this backwards incompatible change required?